### ISSUE TITLE
Is there a way to play on Airplay devices on MacOS?

### ISSUE THREAD

**Issue by dakoller** [ytmdesktop/ytmdesktop#1238]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Feature

I would like to be able to playback youtube music on airplay devices, when being on mac.

**Comment by mingjun97** [comment]
You may select your airplay devices in the control center of your mac. All audio will go to that device in this case. I believe this should apply to ytmd as well.

**Comment by dakoller** [comment]
Right, this is system level setting. If possible, I'd look for being able to configure in app window as well (in a similar way as in iTunes.) 

As far as I can see, you can't select multiple airplay devices in the system settings, but the itunes app allows that.

**Comment by Alipoodle** [comment]
From a quick skim of some of the npm Libraries for connecting to AirPlay Devices, all of them were made 8 years ago and lack support for more modern devices ETC.
Knowing Apple in their choices in stuff~ they're unlikely to let this be an easy task.

Even though this is a Google Product, in a Google made web engine, due to some complications we cannot get Chrome Cast support so easily~
